setup.py

```diff
@@ -96,12 +96,15 @@ def parse_readme():
                 "data/Dockerfiles/dev/*",
                 "data/Dockerfiles/hardhat/*",
                 "data/Dockerfiles/tendermint/*",
+                "data/contracts/*",
+                "data/contracts/component_registry/*",
+                "data/contracts/component_registry/build/*",
+                "data/contracts/registries_manager/*",
+                "data/contracts/registries_manager/build/*",
+                "data/contracts/service_registry/*",
+                "data/contracts/service_registry/build/*",
                 "test_tools/data/*",
             ],
-            "packages": [
-                "valory/contracts/component_registry/*",
-                "valory/contracts/registries_manager/*",
-            ],
```

Comment:
Same as above, not shipping the packages as separate package instead include them in the data dir